pom.xml

```diff
@@ -0,0 +1,218 @@
+	</pluginRepositories>
+
+	<modules>
+		<module>org.faktorips.aggregator</module>
+	</modules>
+
+	<build>
+
+		<pluginManagement>
+			<plugins>
+				<!-- Maven plugins -->
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-release-plugin</artifactId>
+					<version>${maven-release-plugin-version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-checkstyle-plugin</artifactId>
+					<version>${maven-checkstyle-plugin-version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<version>${maven-surefire-plugin-version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-compiler-plugin</artifactId>
+					<version>${maven-compiler-plugin-version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-jar-plugin</artifactId>
+					<version>${maven-jar-plugin-version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-scm-plugin</artifactId>
+					<version>${maven-scm-plugin-version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-javadoc-plugin</artifactId>
+					<version>${maven-javadoc-plugin-version}</version>
+					<configuration>
+						<tags>
+							<tag>
+								<name>apiNote</name>
+								<placement>a</placement>
+								<head>API Note:</head>
+							</tag>
+							<tag>
+								<name>implSpec</name>
+								<placement>a</placement>
+								<head>Implementation Requirements:</head>
+							</tag>
+							<tag>
+								<name>implNote</name>
+								<placement>a</placement>
+								<head>Implementation Note:</head>
+							</tag>
+							<tag>
+								<name>param</name>
+							</tag>
+							<tag>
+								<name>return</name>
+							</tag>
+							<tag>
+								<name>throws</name>
+							</tag>
+							<tag>
+								<name>since</name>
+							</tag>
+							<tag>
+								<name>version</name>
+							</tag>
+							<tag>
+								<name>serialData</name>
+							</tag>
+							<tag>
+								<name>see</name>
+							</tag>
+						</tags>
+					</configuration>
+				</plugin>
+				<!-- tycho plugins -->
+				<plugin>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-maven-plugin</artifactId>
+					<version>${tycho-version}</version>
+					<extensions>true</extensions>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>target-platform-configuration</artifactId>
+					<version>${tycho-version}</version>
+					<configuration>
+						<target>
+							<artifact>
+								<groupId>org.faktorips</groupId>
+								<artifactId>${target-platform}</artifactId>
+								<version>${project.version}</version>
+							</artifact>
+						</target>
```

Comment:
Damit wird jetzt nur noch eine Target-Platform gebaut, oder? Idealerweise würden wir immer noch alle bauen, da diese dann auch vom Integrationtest-Projekt direkt genutzt werden können.

Reply:
Dies legt erstmal nur die default target platfrom fest die für den build benutzt wird.

Man kann dies aber im Aufruf auch überschreiben z.B. `mvn -Dtarget-platform=2021-12 clean install`

Reply:
Ja, aber nachdem die POMs der Target-Platformen gelöscht sind werden diese doch nicht mehr als Artefakte gebaut, die dann der Integrationtest-Build nutzen kann, oder?

Reply:
Doch, die targets werden "pomless" gebaut, d.h. es wird automatisch eine `pom.xml` generiert, so muss man diese nicht per hand anpassen, es reicht dann den Ordner zu kopieren und es in die übergeordnete pom.xml als modul einzutragen. 

Zumindest ist so die Idee :-)